### PR TITLE
[BT-740] Add documentation about adding marking new events as expected in Mixpanel

### PR DESCRIPTION
**Pull request by breilly2**
Just adding some helpful documentation.

### CODE REVIEW

File: src/libs/events.js [hunk 1]
```diff
@@ -3,7 +3,13 @@ import { useEffect } from 'react'
 import { Ajax } from 'src/libs/ajax'
 import { useRoute } from 'src/libs/nav'
 
-
+/*
+ * NOTE: In order to show up in reports, new events MUST be marked as expected in the
+ * Mixpanel lexicon:
+ *   https://mixpanel.com/project/2085496/view/19055/app/lexicon#transformations
```
Comment (nawatts) [inline src/libs/events.js]: This links to the dev Lexicon, but the production Lexicon also needs to be updated and is more important.

File: src/libs/events.js [hunk 2]
```diff
@@ -3,7 +3,13 @@ import { useEffect } from 'react'
...
+ * See our Mixpanel Documentation Guide for more details:
+ *   https://docs.google.com/document/d/11FTFwcSby_Y3syQCvjLz124xgv5LGd0HBhEjup59O7Q
```
Comment (nawatts) [inline src/libs/events.js]: Could this go in the Terra UI wiki?

**Review by nawatts — APPROVED** [review]

File: src/libs/events.js (hunk 1 above)
Comment (breilly2) [reply src/libs/events.js]: I'll update with links to both.

File: src/libs/events.js (hunk 2 above)
Comment (breilly2) [reply src/libs/events.js]: I like that idea. I'll check with the authors of the Google doc to see if that makes sense to them.

**Review by jgainerdewar — APPROVED** [review]

**Pull request merged**